src/test/metersphere/test.yaml

```diff
@@ -0,0 +1,41 @@
+metersphere:
```

Comment:
better to have a switch (off by default) to prevent deployment failure by KS OpenPitrix.

F.Y.I. https://github.com/kubesphere/helm-charts/blob/master/src/test/etcd/templates/tests/test-connection.yaml#L1

Reply:
I have removed it. It's a testing file.